integration/src/test/java/org/mule/test/integration/routing/outbound/UntilSuccessfulRetryExhaustedTestCase.java

```diff
@@ -45,7 +45,7 @@ public void onRetryExhaustedCallExceptionStrategy() throws Exception {
   @Test
   public void onRetryExhaustedErrorTypeMustBeRetryExhausted() throws Exception {
     flowRunner("retryExhaustedCausedByConnectivityError").withPayload("message")
-        .runExpectingException(ErrorTypeMatcher.errorType(MuleErrors.RETRY_EXHAUSTED));
+        .run();
```

Comment:
this doesn't seem right. Is there no failure at all now?

Reply:
The purpose of the test is to validate the logged Message, which is done in the flow configuration (it thows an error if the expected log message is not present).